src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseAssignedUserRolesController.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.hmcts.ccd.v2.external.controller;
+
+import com.google.common.collect.Lists;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.ccd.domain.model.std.CaseAssignedUserRole;
+import uk.gov.hmcts.ccd.domain.service.cauroles.CaseAssignedUserRolesOperation;
+import uk.gov.hmcts.ccd.domain.service.common.UIDService;
+import uk.gov.hmcts.ccd.endpoint.exceptions.BadRequestException;
+import uk.gov.hmcts.ccd.v2.V2;
+import uk.gov.hmcts.ccd.v2.external.resource.CaseAssignedUserRolesResource;
+
+@RestController
+@RequestMapping(path = "/")
+public class CaseAssignedUserRolesController {
+
+    private final UIDService caseReferenceService;
+    private final CaseAssignedUserRolesOperation caseAssignedUserRolesOperation;
+
+    @Autowired
+    public CaseAssignedUserRolesController(UIDService caseReferenceService,
+                                           @Qualifier("authorised") CaseAssignedUserRolesOperation caseAssignedUserRolesOperation) {
+        this.caseReferenceService = caseReferenceService;
+        this.caseAssignedUserRolesOperation = caseAssignedUserRolesOperation;
+    }
+
+    @GetMapping(
+        path = "/case-users"
+    )
+    @ApiOperation(
+        value = "Retrieve Case user roles by Case ID and User ID"
```

Comment:
This should be `Get Case-Assigned Users and Roles`. That is the operation's well-known name in all other references.

Reply:
Changed this